subsys/net/lib/lwm2m/Kconfig

```diff
@@ -205,6 +205,16 @@ config LWM2M_RD_CLIENT_SUPPORT
 	  Client will use registration state machine to locate and connect to
 	  LWM2M servers (including bootstrap server support)
 
+config LWM2M_RD_CLIENT_REUSE_REGISTRATION
+	bool "Reuse an existing LWM2M registration in case of an abort to LWM2M server"
+	default n
```

Comment:
That is not needed, please remove it. Please see https://docs.zephyrproject.org/1.14.0/guides/kconfig/index.html#redundant-defaults